beef

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env ruby
 
+require 'pry-byebug'
```

Comment:
This will prevent BeEF from running unless the user has set the `BEEF_TEST` environment variable before running `bundle install`, or happens to already have the gem installed.

Reply:
Patch removes this.